tests/unit/errors/test_dlrn.py

```diff
@@ -0,0 +1,149 @@
+#! /usr/bin/env python
+"""Test for the DLRN error handlers"""
+
+from unittest.mock import create_autospec
+
+import pytest
+
+from atkinson.errors import BaseError
+from atkinson.errors.dlrn import DlrnFTBFSError
+from atkinson.errors.reporters import BaseReport
+
+
+@pytest.fixture(scope='module')
+def get_config():
+    """ Test config generator """
+    return {'url': 'test/url',
+            'title': 'Test Error'}
+
+
+@pytest.fixture()
+def get_reporter():
+    """ Test reporter mock generator """
+    reporter = create_autospec(BaseReport)
+    return reporter
+
+
+@pytest.fixture()
+def get_report():
+    """ Test reporter mock generator """
+    report = create_autospec(BaseReport)
+    return report
+
+
+def test_is_base(get_config, get_reporter):
+    """
+    Given we have a DlrnFTBFSError instance
+    When we check the type
+    Then we get a BaseError
+    """
+    error = DlrnFTBFSError('Test Release', get_config, get_reporter)
+    assert isinstance(error, BaseError)
```

Comment:
Shouldn't we get an instance of DlrnFTBFSError?  Maybe check issubclass here instead?